modules/ROOT/pages/index.adoc

```diff
@@ -82,6 +82,108 @@ Monitor APIs and endpoints at scheduled intervals. Receive reports about whether
 
 See xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
 
+== Anypoint Monitoring Features
+
+Anypoint Monitoring is the standard method of monitoring Mule application performance, logging, and troubleshooting to reduce the mean time to identify and resolve issues. While all customers receive fundamental monitoring capabilities with their subscription, the Titanium tier offers greater storage, guaranteed resources, and advanced capabilities that allow for greater control and flexibility.
+
+|===
+| Feature | Gold/Platinum subscription features | Titanium subscription features
+
+3+| *Built-in Application Dashboards*
+|More than 80 out-of-the-box charts covering metrics in the following categories: Overview, Inbound, Outbound, Performance, Failures, JVM, Infrastructure
+
+|Yes
+|Yes
+
+|Advanced metrics for Overview, Inbound, Outbound, Performance, Failures, JVM, and Infrastructure charts
+|No
+|Yes
+
+|Flows charts
+|No
+|Yes
+
+|Connectors charts
+|No
+|Yes
+
+|Application Network integration
+|No
+|Yes
+
+|xref:monitoring::performance-and-impact.adoc#data-retention-and-aggregation[Data retention] for more than 30 days
+|No
+|Yes
+
+3+| *Built-in API Dashboards*
+
+|16 out-of-the-box charts for metrics covering the following categories: Overview, Requests, Failures, Performance, Client Application 
+|Yes
+|Yes
+
+
+|xref:monitoring::performance-and-impact.adoc#data-retention-and-aggregation[Data retention] for more than 30 days
+|No
+|Yes
+
+3+| *Custom Dashboards*
+
+|Basic mode
+|Yes
+|Yes
+
+|Advanced mode
+|Yes, with limited functionality
+|Yes
+
+|Advanced mode with additional features
+|No
+|Yes
+
+3+| *Alerts*
+
+|Basic alerts
+|Yes
+|Yes, with xref:performance-and-impact.adoc#data-retention-and-aggregation[higher limits]
+
+|Advanced alerts
+|No
+|Yes
+
+3+| *Log Management*
+
+|Log search
+|No
+|Yes
+
+|Log points
+|No
+|Yes
+
+|Raw data
+|No
+|Yes
+
+3+| *Additional Anypoint Monitoring features*
+
+|Anypoint Monitoring Archive API
+|No
+|Yes
+
+|Functional Monitoring
+|Yes
+|Yes
+
+|Reports
+|No
+|Yes
+
+|Dedicated infrastructure for data processing and storage
+|No
+|Yes
+
+|===
+
 == See Also
```

Comment:
API Functional Monitoring is not included when you use Anypoint Monitoring - might be worth mentioning this to the user in this chart, especially since we don't own it. Link to their docs.